cloudsql/postgres/database-sql/cloudsql_test.go

```diff
@@ -12,14 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package cloudsql
 
 import (
 	"bytes"
 	"net/http/httptest"
 	"os"
 	"strings"
 	"testing"
+
+	cloudsql "github.com/GoogleCloudPlatform/golang-samples/cloudsql/postgres/database-sql"
```

Comment:
hmm isn't this package the package we are in? Why do we need an import here?

Reply:
Another bogus addition. Will fix in the postgres PR.